python/tvm/autotvm/tuner/graph_tuning.py

```diff
@@ -0,0 +1,85 @@
+"""Utilities for tuning a whole graph (a set of tasks)"""
+
+import os
+from . import callback, XGBTuner, GATuner, RandomTuner, GridSearchTuner
+from .. import record, task
+
+def tune_tasks(tasks,
```

Comment:
The function itself seems have nothing to do with whole network, maybe tuner.schedule ?